src/sentry/api/endpoints/organization_pinned_searches.py

```diff
@@ -65,6 +65,18 @@ def put(self, request: Request, organization) -> Response:
                 "query_sort": result["sort"],
             },
         )
+
+        GroupSearchView.objects.create_or_update(
```

Comment:
nit: this isn't really a big deal since this endpoint will be going away after the search views are released, but it's not really clear (to others reading the code) why were are creating these entries here. An explanatory comment would be nice